### DISCUSSION TITLE
Use Local LLM's to translate subtitles

### DISCUSSION THREAD

**Discussion by rowanfuchs** [lingarr-translate/lingarr#19] [Ideas & Feature requests]
Use Local LLM's like Ollama, ChatGPT, Claude e.t.c to translate subtitles

**Comment by bryan065** [comment]
I'm interested as well as I use localai

**Comment by rowanfuchs** [comment]
this functionality has been implemented within the latest release [0.9.4](https://github.com/lingarr-translate/lingarr/releases/tag/0.9.4)
